airflow/contrib/operators/sagemaker_create_training_job_operator.py

```diff
@@ -62,16 +70,23 @@ def __init__(self,
                  sagemaker_conn_id=None,
                  job_name=None,
```

Comment:
Since we already have job_name in training_config, we don't need this job_name here. It increases the risk that users give a different name from config, or forget to give this name here.

We just need to require users providing name in config, that's enough.

Reply:
removed